index.html

```diff
@@ -2,16 +2,45 @@
 <!DOCTYPE html>
 <html xmlns="http://www.w3.org/1999/xhtml">
   <head>
-    <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
-    
+	<meta charset="utf-8">
+	<meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name='viewport' content='width=device-width, initial-scale=1.0, maximum-scale=1'>
+    <meta name="apple-mobile-web-app-capable" content="yes">
+	<meta name="theme-color" content="#0082c9">
+    <link rel="shortcut icon" href="https://nextcloud.com/media/favicon-touch.png" />
+    <link rel="apple-touch-icon-precomposed" href="https://nextcloud.com/media/favicon-touch.png" />
     <title>Nextcloud Documentation</title>
+	<meta name="author" content="Nextcloud">
+	<meta itemprop="description" content="Nextcloud is an open source, self-hosted file share and communication platform. Access  &amp; sync your files, contacts, calendars  &amp; communicate and collaborate across your devices. You decide what happens with your data, where it is and who can access it!">
+	<meta name="description" content="Nextcloud is an open source, self-hosted file share and communication platform. Access  &amp; sync your files, contacts, calendars  &amp; communicate and collaborate across your devices. You decide what happens with your data, where it is and who can access it!">
+
+	<!-- Twitter Card data -->
+	<meta name="twitter:card" content="summary_large_image">
+	<meta name="twitter:site" content="@nextclouders">
+	<meta name="twitter:url" content="https://docs.nextcloud.com">
+	<meta name="twitter:title" content="Nextcloud Documentation">
+	<meta name="twitter:description" content="Nextcloud is an open source, self-hosted file share and communication platform. Access  &amp; sync your files, contacts, calendars  &amp; communicate and collaborate across your devices.">
+
+	<!-- Open Graph data -->
+	<meta property="og:title" content="Nextcloud Documentation" />
+	<meta property="og:type" content="article" />
+	<meta property="og:url" content="http://nextcloud.com" />
+	<meta property="og:description" content="Nextcloud is an open source, self-hosted file share and communication platform. Access  &amp; sync your files, contacts, calendars  &amp; communicate and collaborate across your devices. You decide what happens with your data, where it is and who can access it!" />
+	<meta property="og:site_name" content="Nextcloud" />
+	<meta property="fb:admins" content="1032807203462807" />
+
+	<!-- set images for generic, twitter and Open Graph -->
+	<meta itemprop="image" content="https://docs.nextcloud.com/server/13/user_manual/_images/files_page.png">
```

Comment:
Do you really want to have this in the preview?

![files_page](https://user-images.githubusercontent.com/245432/37452803-eed8b0cc-2835-11e8-94d0-4068e52d975e.png)

Reply:
yes, actually - this is our documentation site and this was the most obvious image that makes that clear. ;-)

Reply:
I spent plenty time finding it :D